### PR TITLE
Bump loaders.gl to v2.0.0-alpha. Use "synchronized" GLTFLoader.

### PR DESCRIPTION
**Pull request by ibgreen**
For #

#### Change List
-

### CODE REVIEW

File: modules/addons/src/gltf/gltf-scenegraph-loader.js [hunk 1]
```diff
@@ -9,40 +8,18 @@ async function parse(data, options, uri, loader) {
...
   const gltfObjects = createGLTFObjects(options.gl, gltf, options);
 
-  if (options.waitForFullLoad) {
-    await waitForGLTFAssets(gltfObjects);
-  }
-
   return Object.assign({gltf}, gltfObjects);
 }
 
-async function waitForGLTFAssets(gltfObjects) {
```
Comment (ibgreen) [inline modules/addons/src/gltf/gltf-scenegraph-loader.js]: Should no longer be needed...

File: modules/addons/src/gltf/gltf-material-parser.js [hunk 2]
```diff
@@ -137,5 +136,11 @@ export default class GLTFMaterialParser {
         ]
       });
     }
+
+    // TODO - placeholder for handling the unlit extension
+    // KHR_materials_unlit extension (postprocessed by GLTFLoader)
+    if (material.unlit) {
```
Comment (tsherif) [inline modules/addons/src/gltf/gltf-material-parser.js]: This seems unrelated to this PR. Not a huge deal, but I'd prefer not to add logic for yet-to-be-defined APIs.
